app/models/form_submission.rb

```diff
@@ -51,6 +51,10 @@ def with_form_types(form_types)
     end
   end
 
+  def benefits_intake_uuid
+    form_submission_attempts.order(created_at: :asc).last&.benefits_intake_uuid
+  end
```

Comment:
I wonder if naming this something like `latest_benefits_intake_uuid` might be more descriptive?

Reply:
I think it's actually more confusing to add this and not use it, on second thought. I will remove it.

Reply:
Actually, I put this back in because we do have several places that call `FormSubmission.benefits_intake_uuid`. An alternate naming would be preferable but I think it involves less intervention if we let `FormSubmission` respond to `#benefits_intake_uuid` like this.